From 653f32d362aa855adf0e32acfa972db1af2103c3 Mon Sep 17 00:00:00 2001
From: Bogdan Togorean <bogdan.togorean@analog.com>
Date: Thu, 16 Feb 2023 16:33:19 +0200
Subject: [PATCH 50/50] drivers: media: i2c: adsd3500.c: Initialize depth bits
 to 0

In order to keep private structure in sync with control data initialize
the number of bits per depth to 0.

Signed-off-by: Bogdan Togorean <bogdan.togorean@analog.com>
---
 drivers/media/i2c/adsd3500.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/adsd3500.c b/drivers/media/i2c/adsd3500.c
index adabf3c67340..aa10a8e17c74 100644
--- a/drivers/media/i2c/adsd3500.c
+++ b/drivers/media/i2c/adsd3500.c
@@ -509,7 +509,7 @@ static const struct v4l2_ctrl_config adsd3500_ctrl_depth_bits = {
 	.id			= V4L2_CID_ADSD3500_DEPTH_BITS,
 	.name		= "Phase / Depth Bits",
 	.type		= V4L2_CTRL_TYPE_INTEGER_MENU,
-	.def		= 4,
+	.def		= 0,
 	.min		= 0,
 	.max		= 6,
 	.menu_skip_mask = 0x02,
-- 
2.39.0

